WebRoot/public/launchZCS.jsp

```diff
@@ -323,13 +323,13 @@
         }
     }
 	function switchToStandardClient() {
-		document.location = window.appContextPath + "/?client=standard";
+		document.location = window.appContextPath + "/?client=modern";
 	}
     killSplashScreenSwitch();
 	<c:set var="enforceMinDisplay" value="${requestScope.authResult.prefs.zimbraPrefAdvancedClientEnforceMinDisplay[0]}"/>
 	<c:if test="${param.client ne 'advanced'}">
 		enforceMinDisplay = ${enforceMinDisplay ne 'FALSE'};
-		unsupported = (screen && (screen.width <= 800 && screen.height <= 600) && !${isOfflineMode}) || (AjxEnv.isSafari && !AjxEnv.isSafari4up);
```

Comment:
if offlinemode is true then also we shouldn't redirect to modern ui

Reply:
Revert change to add isOfflineMode condition